src/dataflow/src/logging/timely.rs

```diff
@@ -122,8 +130,10 @@ pub fn construct<A: Allocate>(
                                 operates_session
                                     .give(&cap, (((event.id, worker), event.name), time_ms, 1));
 
+                                address_validation_session
+                                    .give(&cap, ((event.id, hash_vec(&event.addr)), time_ms, 1));
                                 let address_row = (event.id as i64, worker as i64, event.addr);
-                                addresses_session.give(&cap, (address_row, time_ms, 1));
+                                addresses_session.give(&cap, ((address_row, ()), time_ms, 1));
```

Comment:
DMed you on Slack, but I'll post here too so it doesn't get lost.

I'm confused about why the type of the `addresses` collection is being changed as part of this diff, since you aren't using it anywhere (instead, you're using `address_validation`), unless I'm missing something.

Reply:
In fact, it's an unrelated change. Outputting the data as `((address, ()), time_ms, 1)` instead of `(address, time_ms, 1)` saves us from mapping the data to the former format later on. This is similar to the change in line 162.